go.mod

```diff
@@ -3,57 +3,57 @@ module github.com/openshift-pipelines/opc
 go 1.18
 
 require (
-	github.com/fatih/color v1.13.0 // indirect
+	github.com/fatih/color v1.14.1 // indirect
 	github.com/golang/protobuf v1.5.2 // indirect
 	github.com/google/go-cmp v0.5.9 // indirect
-	github.com/google/go-containerregistry v0.12.1 // indirect
+	github.com/google/go-containerregistry v0.13.0 // indirect
 	github.com/hashicorp/errwrap v1.1.0 // indirect
 	github.com/hashicorp/go-multierror v1.1.1 // indirect
 	github.com/jonboulle/clockwork v0.3.0 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	github.com/tektoncd/pipeline v0.42.0 // indirect
-	github.com/tektoncd/triggers v0.22.0 // indirect
+	github.com/tektoncd/pipeline v0.44.0 // indirect
+	github.com/tektoncd/triggers v0.22.2 // indirect
 	go.opencensus.io v0.24.0 // indirect
 	go.uber.org/atomic v1.10.0 // indirect
 	go.uber.org/multierr v1.9.0 // indirect
 	go.uber.org/zap v1.24.0 // indirect
-	golang.org/x/net v0.5.0 // indirect
-	golang.org/x/sys v0.4.0 // indirect
-	golang.org/x/term v0.4.0 // indirect
+	golang.org/x/net v0.7.0 // indirect
+	golang.org/x/sys v0.5.0 // indirect
+	golang.org/x/term v0.5.0 // indirect
 	golang.org/x/time v0.3.0 // indirect
-	google.golang.org/grpc v1.51.0 // indirect
+	google.golang.org/grpc v1.53.0 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
-	k8s.io/api v0.25.5 // indirect
-	k8s.io/apimachinery v0.25.5 // indirect
-	k8s.io/client-go v0.25.4 // indirect
-	knative.dev/pkg v0.0.0-20221117013641-f5c1a03ab4f1 // indirect
+	k8s.io/api v0.26.1 // indirect
+	k8s.io/apimachinery v0.26.1 // indirect
+	k8s.io/client-go v0.26.1 // indirect
+	knative.dev/pkg v0.0.0-20230125083639-408ad0773f47 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 
 )
 
 require (
-	github.com/openshift-pipelines/pipelines-as-code v0.15.3
+	github.com/openshift-pipelines/pipelines-as-code v0.17.0
 	github.com/spf13/cobra v1.6.1
-	github.com/tektoncd/cli v0.29.1
+	github.com/tektoncd/cli v0.3.1-0.20230228131500-30356f685e13
```

Comment:
Where is this one coming from ? :) current `main` ?

Reply:
yes 
![image](https://user-images.githubusercontent.com/98980/222498752-5566770b-66ea-4127-965f-16cb1df61237.png)

Reply:
I am not doing a replace here so whenever dependabot update to the latest cli release it will hopefully just do the right thing